.travis.yml

```diff
@@ -11,25 +11,30 @@ script:
 
 jobs:
   include:
-  - stage: Unit Test + Code Style Fixing
-    php: 5.3
-  - stage: Unit Test + Code Style Fixing
-    php: 5.4
-  - stage: Unit Test + Code Style Fixing
-    php: 5.5
-  - stage: Unit Test + Code Style Fixing
-    php: 7.0
-  - stage: Unit Test + Code Style Fixing
+  - stage: Unit Tests + Code Style Fixing + Code Coverage
     php: 7.1
-  - stage: Unit Test + Code Style Fixing
-    php: 5.6
+    before_script:
+    - composer require --dev php-coveralls/php-coveralls
+    - composer install
```

Comment:
since you deleted composer-lock file, this actually does nothing, you need to `composer update` instead. 
(why it works currently is because of the require statement above (which I told you to delete :D)

Reply:
I think it will work without the require. It's "install" ?!